timesketch/api/v1/routes.py

```diff
@@ -53,6 +54,7 @@
     (SketchListResource, '/sketches/'),
     (SketchResource, '/sketches/<int:sketch_id>/'),
     (AnalyzerRunResource, '/sketches/<int:sketch_id>/analyzer/'),
+    (AnalyzerPipelineResource, '/sketches/<int:sketch_id>/analyzer/auto_run/'),
```

Comment:
auto_run? I would call it /run/ and accept list of analyzer names as well. If none is provided, run all analyzers, WDYT?

Reply:
this is removed, expanded the use case of the analyzer run resource instead.